cl/search/documents.py

```diff
@@ -89,3 +104,133 @@ def prepare_panel_ids(self, instance):
+        attr="docket.docket_number",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="docket.docket_number", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    docket_slug = fields.KeywordField(attr="docket.slug")
+    duration = fields.IntegerField(attr="duration")
+    download_url = fields.KeywordField(attr="download_url")
+    file_size_mp3 = fields.IntegerField()
+    id = fields.IntegerField(attr="pk")
+    judge = fields.TextField(
+        attr="judges",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(attr="judges", analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    local_path = fields.KeywordField()
+    pacer_case_id = fields.KeywordField(attr="docket.pacer_case_id")
+    panel_ids = fields.ListField(
+        fields.IntegerField(),
+    )
+    sha1 = fields.KeywordField(attr="sha1")
+    source = fields.KeywordField(attr="source")
+    text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    timestamp = fields.DateField()
+
+
+@oral_arguments_index.document
+class AudioDocument(AudioDocumentBase):
+    class Django:
+        model = Audio
+        ignore_signals = True
+
+    def prepare_panel_ids(self, instance):
+        return [judge.pk for judge in instance.panel.all()]
+
+    def prepare_file_size_mp3(self, instance):
+        if instance.local_path_mp3:
+            return deepgetattr(instance, "local_path_mp3.size", None)
+
+    def prepare_local_path(self, instance):
+        if instance.local_path_mp3:
+            return deepgetattr(instance, "local_path_mp3.name", None)
+
+    def prepare_text(self, instance):
+        text_template = loader.get_template("indexes/audio_text.txt")
+        return text_template.render({"item": instance}).translate(null_map)
+
+    def prepare_timestamp(self, instance):
+        return datetime.utcnow()
+
+
+class PercolatorField(fields.DEDField, Percolator):
+    pass
+
+
+@oral_arguments_percolator_index.document
+class AudioPercolator(AudioDocumentBase):
+    rate = fields.KeywordField(attr="rate")
+    percolator_query = PercolatorField()
+
+    class Django:
+        model = Alert
+        ignore_signals = True
+
+    def prepare_timestamp(self, instance):
+        return datetime.utcnow()
+
+    def prepare_percolator_query(self, instance):
+        qd = QueryDict(instance.query.encode(), mutable=True)
+        search_form = SearchForm(qd)
+        if not search_form.is_valid():
+            return None
+
+        cd = search_form.cleaned_data
+        search_query = AudioDocument.search()
+        (
+            query,
+            total_query_results,
+            top_hits_limit,
```

Comment:
Solved, now these vars are not returned, since I created a new method `build_es_base_query` which doesn't return them. Also this method is used for other basic queries like the Alert estimation query.